### DISCUSSION TITLE
Exclude some files from what deploy copies?

### DISCUSSION THREAD

**Discussion by jasonkuhrt** [pnpm/pnpm#5164] [Q&A] [answered]
Currently `deploy` makes a copy of the project directory contents. I'd like to exclude `src`. Currently I am just using `rm -rf ...` after running the deploy command. Is there a better way?

**Comment by zkochan** [comment, marked as answer]
Just add a `files` field to the package.json of the deployed project. pnpm will only copy the files and directories from that list.
